tests/PactTestsSpec.hs

```diff
@@ -93,7 +93,7 @@ badErrors = M.fromList
   ,(pfx "bad-parens.repl",
     "error: expected")
   ,(pfx "bad-import-unimported-reference.repl",
-    "Cannot resolve")
+    "Expected def")
```

Comment:
This must be related to the change to the assoc. repl script. I don't understand the change.